cmd/hostd/run.go

```diff
@@ -449,6 +449,9 @@ func runRootCmd(ctx context.Context, cfg config.Config, walletKey types.PrivateK
 		api.WithSQLite3Store(store),
 	}
 	if !cfg.Explorer.Disable {
+		if cfg.Explorer.URL == "" {
+			cfg.Explorer.URL = "https://api.siascan.com"
```

Comment:
I would've probably defaulted in `LoadFile` and added a comment why the default is necessary. If you say "migration" do you refer to `updateConfigV112`? If so, shouldn't we check for empty explorer URL in that function as well?

Reply:
Yeah, it seems to be an issue configs migrated with `updateConfigV112`. The field gets explicitly set to "",  which ends up overriding the default when in reality we wanted it unset.

Reply:
Im going to close this for now and consider restructuring the way we handle default values